### DISCUSSION TITLE
封面

### DISCUSSION THREAD

**Discussion by xiatianPE** [Qexo/Qexo#415] [Ideas]
能否单独做个文章封面功能，每次要配置其他参数属实有一点麻烦

**Comment by am-abudu** [comment]
右侧侧边栏可编辑。点击画笔按键即可

**Reply by xiatianPE** [reply]
突然发现了一个问题就是没有封面上传功能，那该如何弄？

**Reply by am-abudu** [reply]
你可以在左边的编辑栏上传，然后复制链接到输入框内

**Reply by xiatianPE** [reply]
为什么？编辑栏上有时候是这样
![51F06C5E-8EBE-41C0-94D0-4C34A5318DB6](https://github.com/Qexo/Qexo/assets/121849812/335bcfb2-4fba-40f3-8620-281991b604b2)
有时候又是这样
![51CD0C5E-7C73-45AA-AC0F-99FE969B09F2](https://github.com/Qexo/Qexo/assets/121849812/639eb7a0-f1b6-4584-8b90-a5b476bad75e)
这是什么情况？

**Reply by am-abudu** [reply]
这是根据你的设备类型判断的

**Reply by xiatianPE** [reply]
那能否固定？我比较喜欢第一种

**Reply by am-abudu** [reply]
若是固定的话，第一种在小屏幕设备（如手机）会及其难受
